store/copr/main_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package copr
+
+import (
+	"testing"
+	"time"
+
+	"github.com/pingcap/tidb/util/testbridge"
+	"go.uber.org/goleak"
+)
+
+type main struct {
+	m *testing.M
+}
+
+func (m *main) Run() int {
+	c := m.m.Run()
+	// In order for MVCCLevelDB to close, we need to wait one second
+	time.Sleep(time.Second)
+	return c
+}
```

Comment:
BTW, this is a nice pattern I'd like to learn. Thanks!

Reply:
You may be interested in https://github.com/pingcap/tidb/pull/27743/commits/9303e335922f22a02dfc82060119c95c1a11325f#diff-c82f35d024189fbb1b0627d10999e1c601d107cb051e23ceccf89002da395af5

Reply:
Nice work.